### PR TITLE
Http link community

### PR DESCRIPTION
**Pull request by aljesusg**
Add link http to groups for websites, the icon is 
[HTTP Material Icon](https://material.io/icons/#ic_http)

We wanna have in database group.web field for GDGGranada => gdggranada.com and GDGMadrid => gdgmadrid.com

### CODE REVIEW

**Comment by aljesusg** [comment]
@abdonrd Review this please

File: src/app-icons.html [hunk 1]
```diff
@@ -14,13 +14,14 @@
...
       <g id="googlePlus"><path d="M7.6 11c0 0.9 0 1.7 0 2.6 1.5 0 2.9 0 4.4 0 -0.6 3.2-5 4.3-7.3 2.2 -2.4-1.9-2.3-5.9 0.2-7.6 1.7-1.4 4.2-1.1 5.9 0.1 0.7-0.6 1.3-1.3 1.9-2 -1.5-1.2-3.2-2-5.1-1.9C3.7 4.3 0.1 7.7 0 11.7c-0.2 3.2 1.9 6.4 4.8 7.5 2.9 1.2 6.8 0.4 8.7-2.3 1.3-1.7 1.5-3.9 1.4-5.9C12.4 11 10 11 7.6 11zM21.8 11c0-0.7 0-1.5 0-2.2 -0.7 0-1.5 0-2.2 0 0 0.7 0 1.5 0 2.2 -0.7 0-1.5 0-2.2 0 0 0.7 0 1.5 0 2.2 0.7 0 1.5 0 2.2 0 0 0.7 0 1.5 0 2.2 0.7 0 1.5 0 2.2 0 0-0.7 0-1.5 0-2.2 0.7 0 1.5 0 2.2 0 0-0.7 0-1.5 0-2.2C23.2 11 22.5 11 21.8 11z"/></g>
+      <g id="web"><path d="M4.5 11h-2V9H1v6h1.5v-2.5h2V15H6V9H4.5v2zm2.5-.5h1.5V15H10v-4.5h1.5V9H7v1.5zm5.5 0H14V15h1.5v-4.5H17V9h-4.5v1.5zm9-1.5H18v6h1.5v-2h2c.8 0 1.5-.7 1.5-1.5v-1c0-.8-.7-1.5-1.5-1.5zm0 2.5h-2v-1h2v1z"/><path d="M24 24H0V0h24v24z" fill="none"/></g>
```
Comment (abdonrd) [inline src/app-icons.html]: Can you update the icon to the ["`http`" `iron-icon`](https://github.com/PolymerElements/iron-icons/blob/master/iron-icons.html#L178)?

``` html
<g id="http"><path d="M4.5 11h-2V9H1v6h1.5v-2.5h2V15H6V9H4.5v2zm2.5-.5h1.5V15H10v-4.5h1.5V9H7v1.5zm5.5 0H14V15h1.5v-4.5H17V9h-4.5v1.5zm9-1.5H18v6h1.5v-2h2c.8 0 1.5-.7 1.5-1.5v-1c0-.8-.7-1.5-1.5-1.5zm0 2.5h-2v-1h2v1z"/></g>
```

Thanks!

File: src/app-icons.html [hunk 2]
```diff
@@ -14,13 +14,14 @@
...
-    </defs>
+  </defs>
```
Comment (abdonrd) [inline src/app-icons.html]: Get back the indent, please. 😅

File: src/pages/groups/group-item.html [hunk 3]
```diff
@@ -65,6 +65,16 @@
                   hidden$="[[!group.googlePlus]]"></paper-icon-button>
             </a>
           </template>
+          <template is="dom-if" if="[[group.web]]">
+            <a href$="[[group.web]]" target="_blank"
+                tabindex="-1"
+                title$="[[group.web]]"
+                aria-label$="[[group.web]]">
+              <paper-icon-button
+                  icon="web"
```
Comment (abdonrd) [inline src/pages/groups/group-item.html]: Update `icon="web"` to `icon="http"`, please. :)

**Review by abdonrd — CHANGES_REQUESTED** [review]
Thanks for the PR!
Now I will add the links in Firebase.

File: src/app-icons.html (hunk 1 above)
Comment (aljesusg) [reply src/app-icons.html]: Ok, I didn't see http icon in ironset !!! anyway maybe is better have the www globe icon instead http

Comment (abdonrd) [reply src/app-icons.html]: What is the www globe icon? Link please! :)

Maybe you want `editor:insert-link`? Check it in the [iron-icons demo](https://elements.polymer-project.org/elements/iron-icons?view=demo:demo/index.html).

**Comment by aljesusg** [comment]
@abdonrd Rdy for merge after build

**Review by abdonrd — APPROVED** [review]

**Comment by abdonrd** [comment]
Thanks @aljesusg! 🎉

**Comment by abdonrd** [comment]
Working at develop site: https://gdg-es-develop.firebaseapp.com/groups

**Pull request merged**